src/Trixi.jl

```diff
@@ -37,9 +37,10 @@ using OffsetArrays: OffsetArray, OffsetVector
 using RecipesBase
 using Requires
 @reexport using StaticArrays: SVector
-using StaticArrays: MVector, MArray, SMatrix
+using StaticArrays: MVector, MMatrix, MArray, SMatrix
```

Comment:
Why add `MMatrix`? A quick look did not reveal where this was used, but maybe I missed it...

Reply:
Leftover from developing this PR and comparing lots of options... (44abf1d9)